build.gradle

```diff
@@ -1,11 +1,18 @@
 buildscript {
+    ext.kotlin_version = '1.3.72'
+
     repositories {
         jcenter()
         google()
+        maven {
+            url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
```

Comment:
We shouldn't need this.

Reply:
In a future beta version it can come in handy, so no need to be there but doesn't do any damages if there are.